docs/integration.rst

```diff
@@ -1,6 +1,9 @@
 Integrating django-helpdesk into your application
 -------------------------------------------------
```

Comment:
Maybe make this line with equals to mark a difference between title and subtitle